.github/workflows/client-build.yml

```diff
@@ -7,31 +7,52 @@ on:
       - '*'
   pull_request:
     types: [opened, reopened, synchronize]
+
+env:
+  cwd: ${{github.workspace}}/packages/client
+
+defaults:
+  run:
+    working-directory: packages/client
+
 jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [10, 12, 13]
+        node-version: [12]
```

Comment:
This is fine for this PR but can we open an issue tracker here where we will run these node tests in the nightly tests or the master branch tests?

Reply:
agreed. and should we run the client karma browser tests on PR? they can tend to be problematic on a per PR basis. right now it's pretty quick as i believe it just executes the `chain.spec.ts` tests, but i'll add it in my TODO to slowly start adding more files there.

Reply:
Will leave the browser tests for now, feel free to remove from CI when you think they are getting problematic to run.